src/Compilers/CSharp/Test/Emit/Microsoft.CodeAnalysis.CSharp.Emit.UnitTests.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.CSharp.UnitTests</RootNamespace>
-    <TargetFrameworks>netcoreapp3.1;net472</TargetFrameworks>
+    <TargetFrameworks>net5.0;net472</TargetFrameworks>
```

Comment:
Just to confirm, we're intentionally keeping our other test projects targeting netcoreapp3.1?

Reply:
I suspect it is @jaredpar's intention to switch them all at some point.

---
In reply to: [460321746](https://github.com/dotnet/roslyn/pull/46285#discussion_r460321746) [](ancestors = 460321746)